aten/src/ATen/native/Activation.cpp

```diff
@@ -576,8 +577,9 @@ inline void _rrelu_with_noise_train(
     const Scalar& lower_,
     const Scalar& upper_,
     c10::optional<Generator> generator) {
-  scalar_t lower = lower_.to<scalar_t>();
-  scalar_t upper = upper_.to<scalar_t>();
+  using acc_t = at::opmath_type<scalar_t>;
```

Comment:
nit: here (and in all your half PRs) please use `opmath_t`, not `acc_t`

Reply:
updated!